src/node/commands/simulate.ts

```diff
@@ -1,7 +1,13 @@
+import { cpus } from 'os';
+import { fork } from 'child_process';
```

Comment:
I think I'll have to resolve this to a custom stubbed module if I want to be able to run this simulator in the web.